psakefile.ps1

```diff
@@ -103,9 +113,15 @@ task Bump {
   exec { npm version --no-git-tag-version $packageVersion }
 }
 
+task Npm -precondition { !(Test-Path (Join-Path $PSScriptRoot 'node_modules')) }{
+  exec { npm install }
+}
+
 task Vendor -depends VendorEditorServices, VendorEditorSyntax, VendorCytoscape
 
-task Build -depends Clean, Vendor, CompileTypeScript
+task BuildOrig -depends Clean, Npm, Vendor, CompileTypeScript
```

Comment:
Do we need to keep BuildOrig?

Reply:
Thanks. I had kept it during testing with intention to remove